pkg/k8s/k8s_rolebinding.go

```diff
@@ -0,0 +1,60 @@
+package k8s
+
+import (
+	"context"
+	"fmt"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"time"
+)
+
+// CreateRoleBinding creates a roleBinding
+func CreateRoleBinding(name, namespace string, labels map[string]string, role, serviceAccount string) error {
+
+	roleBinding := &rbacv1.RoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+			Labels:    labels,
+		},
+		RoleRef: rbacv1.RoleRef{
+			Kind: "Role",
+			Name: role,
+		},
+		Subjects: []rbacv1.Subject{
+			{
+				Kind:      "ServiceAccount",
+				Name:      serviceAccount,
+				Namespace: namespace,
+			},
+		},
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), 20*time.Second)
```

Comment:
you can make 20*time.Second a const and use it twice

Reply:
https://github.com/celestiaorg/knuu/issues/115